src/main/java/com/example/spottyv2/Entities/SpotifyUser.java

```diff
@@ -0,0 +1,13 @@
+package com.example.spottyv2.Entities;
+
+public class SpotifyUser extends User{
+
+    /**
+     * Construct a User object, with the following parameter:
+     *
+     * @param userID        Username of the User
+     */
+    public SpotifyUser(String userID) {
+        super(userID, false);
```

Comment:
Once we merge my commit to main we can get the userId from the spotifyApi directylu